src/views/editor/text.rs

```diff
@@ -359,50 +356,64 @@ pub trait Styling {
     }
 }
 
-pub fn default_light_color(color: EditorColor) -> Color {
+pub fn default_light_theme(mut style: EditorCustomStyle) -> EditorCustomStyle {
```

Comment:
Name for light variant was changed but name of dark variant wasn't. Is there some reason for the difference or why was it changed?